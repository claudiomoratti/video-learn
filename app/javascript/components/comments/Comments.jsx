import React from 'react'

const Comments = ({greeting, lesson_id}) => {

  return (
    <div>
      <span>{greeting}</span>
      <span>{lesson_id}</span>
    </div>
  )
}

export default Comments;
