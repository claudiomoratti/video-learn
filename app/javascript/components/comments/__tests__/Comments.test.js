import React from 'react';
import renderer from "react-test-renderer";
import Comments from "../Comments";

describe('Comments', () => {
  it('Renders default Comments', () => {
    const component = renderer.create(
      <Comments greeting={'Ciao :)'} lesson_id={5}/>
    );

    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
