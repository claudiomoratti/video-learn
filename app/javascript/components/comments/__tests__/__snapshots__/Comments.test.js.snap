// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments Renders default Comments 1`] = `
<div>
  <span>
    Ciao :)
  </span>
  <span>
    5
  </span>
</div>
`;
